### ISSUE TITLE
Feature 'minimum instances' is not supported in the declared launch stage

### ISSUE THREAD

**Issue by mattes** [google-github-actions/deploy-cloudrun#35]
I'm getting this error and can't deploy:

![image](https://user-images.githubusercontent.com/1240531/107094389-c6653980-67bb-11eb-8bba-86676e7be673.png)

```
Error: The feature 'minimum instances' is not supported in the declared launch stage on resource xxxx. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.
```

Tried with `v0.3.0` and latest` main`.

**Comment by mattes** [comment]
The `bug` label was added automatically. :-) The problem I'm running into might be a user error.

**Comment by averikitsch** [comment]
Hi @mattes, how are you using the action, ie are you using the service + image inputs or the metadata input? This may be related to an issue I am currently working on.

**Comment by mattes** [comment]
Sorry, should've posted this earlier. Here is the Github Action spec:

```
    - name: Deploy to Cloud Run
      if: github.ref == 'refs/heads/master'
      uses: google-github-actions/deploy-cloudrun@main
      with:
        image: gcr.io/xxxx/xxxx:${{github.run_number}}-${{github.sha}}
        service: xxxx 
        credentials: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
```

**Comment by amctavish** [comment]
I'm having the same issues but I'm using `google-github-actions/setup-gcloud` as my auth method:
```
- name: Deploy to Cloud Run
  id: deploy
  uses: google-github-actions/deploy-cloudrun@v0.3.0
  with:
    image: eu.gcr.io/${{ secrets.GCP_PROJECT_ID }}/${{ env.SERVICE }}:${{ github.sha }}
    region: europe-west3
    service: ${{ env.SERVICE }}
```

**Comment by averikitsch** [comment]
I apologize for the delay. This should be fixed on the main branch or v0.4.0:
```
uses: google-github-actions/deploy-cloudrun@v0.4.0
```

Please reopen if you are still experiencing this issue

**Comment by mattes** [comment]
Thanks for fixing!
